t/plugin/authz-keycloak.t

```diff
@@ -137,7 +224,7 @@ done
                             "authz-keycloak": {
                                 "token_endpoint": "http://127.0.0.1:8090/auth/realms/University/protocol/openid-connect/token",
                                 "permissions": ["course_resource#view"],
-                                "audience": "course_management",
```

Comment:
Maybe we can leave `audience` unchanged in one or two cases so that we know the old configuration is still compatible?

Reply:
Sure, I can add some cases that use it.